api/models/slideshow.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const slide = new Schema({
+    URL: { type: String, default: "https://i.imgur.com/3MlKktU.jpg" },
```

Comment:
change `URL` to `data` since in the future this might hold a reference to the content instead of a url